openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/weighting/MaximumSpeedWeighting.java

```diff
@@ -0,0 +1,116 @@
+/*  This file is part of Openrouteservice.
+ *
+ *  Openrouteservice is free software; you can redistribute it and/or modify it under the terms of the
+ *  GNU Lesser General Public License as published by the Free Software Foundation; either version 2.1
+ *  of the License, or (at your option) any later version.
+
+ *  This library is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *  See the GNU Lesser General Public License for more details.
+
+ *  You should have received a copy of the GNU Lesser General Public License along with this library;
+ *  if not, see <https://www.gnu.org/licenses/>.
+ */
+package org.heigit.ors.routing.graphhopper.extensions.weighting;
+
+import com.graphhopper.routing.profiles.DecimalEncodedValue;
+import com.graphhopper.routing.util.FlagEncoder;
+import com.graphhopper.routing.util.HintsMap;
+import com.graphhopper.routing.weighting.Weighting;
+import com.graphhopper.util.EdgeIteratorState;
+import com.graphhopper.util.Parameters.Routing;
+
+/**
+ * This class creates the weighting for the routing according to the maximum speed set by user.
+ *
+ * @author Athanasios Kogios
+ */
+
+public class MaximumSpeedWeighting implements Weighting {
+    protected final static double SPEED_CONV = 3.6; //From km/h to m/s.
```

Comment:
Use a more expressive name instead of a comment.

Reply:
Done.